cmd/main.go

```diff
@@ -199,14 +236,15 @@ func main() {
 		options = append(options, thriftcheck.WithLogger(logger))
 	}
 
-	if len(flag.Args()) == 0 {
+	filenames := flag.Args()
```

Comment:
nit: I think this should be named `paths` now, too. Or we can just leave it as `flag.Args()`; that isn't as confusing as I had originally thought.

Reply:
fixed it